tests/MongoDB.Driver.Tests/IAggregateFluentExtensionsTests.cs

```diff
@@ -198,6 +198,35 @@ public void Lookup_should_generate_the_correct_group_when_using_lambdas()
             AssertLast(subject, expectedLookup);
         }
 
+        [Fact]
+        public void Lookup_expressive_should_generate_the_correct_group_when_using_BsonDocument()
```

Comment:
Why does this say "...correct_group..."?

Seems like it should be "...correct_lookup..." or "...correct_stage...".

Reply:
I just copypasted the previous lookup tests names with adding `expressive` word. I will rename my tests